tools/proto_format/proto_sync.py

```diff
@@ -162,8 +162,13 @@ def get_destination_path(src):
     if len(matches) != 1:
         raise RequiresReformatError(
             "Expect {} has only one package declaration but has {}".format(src, len(matches)))
-    return pathlib.Path(get_directory_from_package(
+    dst_path = pathlib.Path(get_directory_from_package(
         matches[0])).joinpath(src_path.name.split('.')[0] + ".proto")
+    # contrib API files have the standard namespace but are in a contrib folder for clarity.
```

Comment:
I'd prefer tooling that enforces that everything in `contrib/` is by default `v3alpha`. Then we can explicitly allowlist the few exceptions (grandfathered or agreed on API shepherd coverage). This will prevent accidental leakage of contrib APIs into stable.